internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,53 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
-}
+	times := strings.Split(data, ",")
+	if len(times) != 2 {
+		return 0, 0, errors.New("Insufficient data")
+	}
 
+	steps, err := strconv.Atoi(times[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("conversion error: %w", err)
+	}
+	if steps <= 0 {
+		return 0, 0, errors.New("The number of steps must be greater than zero")
+	}
+
+	duration, err := time.ParseDuration(times[1])
+	if err != nil {
+		return 0, 0, errors.New("Invalid duration format")
```

Comment:
Ожидалось, что ты поправишь везде
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
И в других файлах тоже